### ISSUE TITLE
is this compatible with pytorch models?

### ISSUE THREAD

**Issue by ethansmith2000** [SimonBlanke/Gradient-Free-Optimizers#37]

**Comment by SimonBlanke** [comment]
Hello @ethansmith2000,

yes you can do hyperparameter optimization of pytorch models in Gradient-Free-Optimizers. You can do it by returning a score (or negative loss) after the finished fit of the model in the objective-function. There is no pytorch example in this repository, yet.
There is an example for this in the [Hyperactive repository](https://github.com/SimonBlanke/Hyperactive/blob/master/examples/tested_and_supported_packages/pytorch_example.py). You can just copy the objective function (called "pytorch_cnn" in the example) and the search-space from the example, but be aware that the rest of the Hyperactive-package works differently.
